### DISCUSSION TITLE
LUCI界面编译出来默认是5秒刷新一次，如何修改为1秒刷新一次

### DISCUSSION THREAD

**Discussion by rianjskis** [coolsnowwolf/lede#12097] [Q&A / 快速问答]
就是这个界面![Screenshot_20240425_103350.jpg](https://github.com/coolsnowwolf/lede/assets/86937445/757ab391-e36a-4fe0-993f-3427f3cafe71)

**Comment by WYC-2020** [comment]
改这里
![image](https://github.com/coolsnowwolf/lede/assets/34025388/a0cbc3b9-32e7-4a81-98c8-ee4be67f109c)
![image](https://github.com/coolsnowwolf/lede/assets/34025388/3754734d-259c-4ebc-b74e-97dfa70f4a78)
  5改成1

**Reply by rianjskis** [reply]
感谢告知，已经编译刷写好了可以在ssh中修改吗,可以的话应该修改哪个文件

**Reply by WYC-2020** [reply]
/etc/index.htm

**Reply by rianjskis** [reply]
> /etc/index.htm

感谢，完美，老哥请问如果是官方固件您知道如何修改吗，我编译有两种，麻烦您了

**Reply by WYC-2020** [reply]
不清楚从不用官方的 那玩意js用起就不舒服

**Reply by rianjskis** [reply]
> 不清楚从不用官方的 那玩意js用起就不舒服

好的谢谢
